src/ui/layout/message-layout-error.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import ColLayout from "./col-layout.svelte";
+  import RowLayout from "./row-layout.svelte";
+  import Typography from "../atoms/typography.svelte";
+  import { onMount } from "svelte";
+  const { title = null, icon = null, message } = $props();
+
+  const MAX_SINGLE_LINE_PIXEL_HEIGHT = 32;
+
+  let titleElement: HTMLElement | null = $state(null);
+  let isTitleWrapped = $state(false);
+  let resizeObserver: ResizeObserver;
+
+  onMount(() => {
+    if (titleElement) {
+      resizeObserver = new ResizeObserver(() => {
+        isTitleWrapped =
+          (titleElement?.offsetHeight ?? 0) > MAX_SINGLE_LINE_PIXEL_HEIGHT;
+      });
+      resizeObserver.observe(titleElement);
+    }
+
+    return () => {
+      resizeObserver?.disconnect();
+    };
+  });
```

Comment:
If the line height of the title exceeds 32px we assume it has wrapped to two+ lines, in which case we want to use the larger icon.